### ISSUE TITLE
OISD NSFW not working

### ISSUE THREAD

**Issue by anxiouslelimo** [serverless-dns/blocklists#164]
Even after enabling oisd nsfw blocklist adult sites are loading. Verified that the sites are included in oisd list.

**Comment by ignoramous** [comment]
Thanks for the confirmation! (:

I remember looking at this a week ago and wondering where the issue really was. Looks like it isn't present anymore. One of the recent changes must have fixed this.

**Comment by ignoramous** [comment]
Every 8th day of the month (automated):
- For apps: 1, 8, 16, 24 ([ref](https://github.com/serverless-dns/blocklists/blob/092017acbe973646de386207492bed03cada1f13/.github/workflows/createUploadBlocklistFilter.yml#L6))
- For Max: 2, 9, 17, 25 ([ref](https://github.com/serverless-dns/serverless-dns/blob/47a9bf1aafd53c37656addc06ad452261b255e46/.github/workflows/fly.yml#L7))
- For Sky: 3, 10, 18, 26 ([ref](https://github.com/serverless-dns/serverless-dns/blob/47a9bf1aafd53c37656addc06ad452261b255e46/.github/workflows/cf.yml#L6))

**Comment by ignoramous** [comment]
`sky` is subject to heavy DoS attacks, and it may behave differently. We are working on a new architecture to bring down the costs for `sky`.

As for `max`, it continues to function but our infrastructure provider has trouble keeping up the amount of requests we are sending their way. Of late, automated blocklist updates have been failing (again), too. I'll look into them today and get back to you.

**Comment by ignoramous** [comment]
Since Nov 18, We're locked out of some of the Cloudflare services due to 'trust and safety' issue for which we have received no answer, as of today. Blocklist builds are working, but the uploads and downloads to Cloudflare R2 (blob store) is not. The alternative is to use S3, but we moved to R2 because hosting on S3 costs us $500/mo compared to $5/mo on R2.

**Comment by ignoramous** [comment]
The blocklist updates are up and working (for now); the latest blocklist version is `1701089546918`.

**Comment by ignoramous** [comment]
> @ignoramous Still sites unblocked in max

Thanks.

Our weekly automated deploys (which also updates blocklists) to our resolvers on `Fly.io` (`max`) failed ([ref](https://github.com/serverless-dns/serverless-dns/actions/runs/6987869911/job/19014852643)). I've re-ran it so it'd pickup the latest blocklists, just now.

**Comment by ignoramous** [comment]
In case you didn't know, anyone can check the last update time and search inside blocklists for domains on our website; ex: https://rethinkdns.com/search?q=segment.io

The next auto-update is scheduled for 1 Dec at ~22:00 (UTC).

**Comment by ignoramous** [comment]
*OISD (Adult)* is loading just fine.

*Sky* is facing huge amounts of traffic ($8000+ per month) and so it is run in "restricted" mode to keep the costs in control, as a result of which not ALL blocklists are applied (only a few good trackerware/spyware/family/malware ones are). *Max* however continues to run at full-capability.

We'll probably get *Sky* sorted sometime later this year (but that's a separate issue unrelated to this one). Closing... (but feel free to re-open in case the original issue of *OISD (Adult)* not working is still a problem).
